tests/fermi/test_bravyi_kitaev.py

```diff
@@ -847,7 +847,7 @@ def test_bravyi_kitaev_fermi_word_ps(fermionic_op, n_qubits, result):
     assert qubit_op == expected_op
 
 
-@pytest.mark.usefixtures("use_legacy_opmath")
+@pytest.mark.usefixtures("legacy_opmath_only")
```

Comment:
Again, this test is an exact duplicate of the one above. The parameters that it uses are also exact duplicates of the parameters used above, but created in a legacy opmath context. This test along with the parameters it uses can all be deleted.